src/pmpo_mesh.hpp

```diff
@@ -59,20 +78,21 @@ class Mesh {
           int numElms,
           DoubleVec3dView vtxCoords,
           IntVtx2ElmView elm2VtxConn,
-          IntElm2VtxView vtx2ElmConn,
           IntElm2ElmView elm2ElmConn ):
           meshType_(meshType),
           geomType_(geomType),
           sphereRadius_(sphereRadius),
           numVtxs_(numVtxs),
           numElms_(numElms),
-          vtxCoords_(vtxCoords),
           elm2VtxConn_(elm2VtxConn),
-          vtx2ElmConn_(vtx2ElmConn),
           elm2ElmConn_(elm2ElmConn){
-            vtxVel_ = DoubleVec2dView("vtxVelocity",numVtxs);
+            meshEdit_ = true;
+            setMeshVtxBasedFieldSize(numVtxs);
```

Comment:
don't pass `numVtxs` here - it is already set via the initializer list

Reply:
Same commit as above: https://github.com/SCOREC/polyMPO/commit/395733aa690fc199e6af8cbe9c53a057e56c9c98